doc/docs/master/deploy-manage/manage/s3gateway/configure-s3client.md

```diff
@@ -0,0 +1,142 @@
+# Configure the S3 client
+
+Before you can work with the S3 gateway, configure your S3 client
+to access Pachyderm. Complete the steps in one of the sections below that
+correspond to your S3 client.
+
+## Configure MinIO
+
+If you are not using the MinIO client, skip this section.
+
+To install and configure MinIO, complete the following steps:
+
+1. Install the MinIO client on your platform as
+described on the [MinIO download page](https://min.io/download#/macos).
+
+1. Verify that MinIO components are successfully installed by running
+the following command:
+
+   ```bash
+   minio version
+   mc version
+   ```
+
+   **System Response:**
+
+   ```bash
+   Version: 2019-07-11T19:31:28Z
+   Release-tag: RELEASE.2019-07-11T19-31-28Z
+   Commit-id: 31e5ac02bdbdbaf20a87683925041f406307cfb9
+   ```
+
+1. Set up the MinIO configuration file to use the `30600` port for your host:
+
+   ```bash
+   vi ~/.mc/config.json
+   ```
+
+   You should see a configuration similar to the following:
+
+   * For a minikube deployment, verify the
+   `local` host configuration:
+
+     ```bash
+     "local": {
+               "url": "http://localhost:30600",
+               "accessKey": "YOUR-PACHYDERM-AUTH-TOKEN",
+               "secretKey": "YOUR-PACHYDERM-AUTH-TOKEN",
+               "api": "S3v4",
+               "lookup": "auto"
+            },
+     ```
+
+     Set the access key and secret key to your
+     Pachyderm authentication token. If authentication is not enabled
+     on the cluster, both parameters must be empty strings.
```

Comment:
if auth is not enabled, they can be any values. minio might also be able to handle those keys not being in the config, I'm not sure.